snapshots/snapshots-processor/src/main/kotlin/com/emergetools/snapshots/processor/PreviewProcessor.kt

```diff
@@ -45,26 +45,27 @@ class PreviewProcessor(
     val previewFunctions = resolver
       .getSymbolsWithAnnotation(COMPOSE_PREVIEW_ANNOTATION_NAME)
       .filterIsInstance<KSFunctionDeclaration>()
+      .toList()
```

Comment:
The toList is an important change. Previously, we were dealing with a sequence, but it was introducing some very strange issues where moving files were somehow being done in parallel with generation, leading to some strange failures. Converting to list ensures we avoid this.